api/src/modules/characters/controllers/character.ctrl.spec.ts

```diff
@@ -0,0 +1,115 @@
+import assert from 'assert';
+
+import status from 'http-status-codes';
+
+import testApi from '../../../shared/spec/api';
+import { createTestUserAndLogin } from '../../../shared/spec/db';
+import {
+  createPostTestForGuardAgainstUnlogin,
+  createGetTestForGuardAgainstUnlogin,
+} from '../../../shared/spec/security';
+
+type createCharReqType = {
+  name: string;
+  race: string;
+};
+
+function createCharacter(
+  token: string,
+  creationPayload: createCharReqType = {
+    name: 'Some name',
+    race: 'elf',
+  }
+) {
+  return testApi(token)
+    .post('/users/current/characters')
+    .send(creationPayload);
+}
```

Comment:
I would like to avoid such helper function if they do not provide real value (e.g. if there are multiple steps, few calls etc. it's ok, but here it's very simple call).

what is your gain now? you just "hide" default payload in test. and hide the action nature - `createCharacter` is not obvious that you making a call to the api - u could use context, create it directly by typerom etc. etc.

I would just remove the function. but if u really like, do not provide default character, or at least call make default a human.